vault/testing.go

```diff
@@ -266,6 +279,60 @@ func testTokenStore(t testing.TB, c *Core) *TokenStore {
 	return ts
 }
 
+// TestIdentityStoreWithGithubAuth returns an instance of identity store which
+// is mounted by default. This function also enables the github auth backend to
+// assist with testing identities and entities what require an valid mount ID
+// of an auth backend.
+func TestIdentityStoreWithGithubAuth(t testing.TB) *identityStore {
```

Comment:
Current tests only required a valid auth mount so that when identity store could validate the supplied mount_path and convert it into its corresponding mount_id. So, it doesn't matter if I use LDAP or Github in this respect.